examples/table/table/app.py

```diff
@@ -67,10 +67,16 @@ def startup(self):
         self.label_table2 = toga.Label('Try multiple row selection.', style=Pack(flex=1, padding_left=5))
         labelbox = toga.Box(children=[self.label_table1, self.label_table2], style=Pack(flex=0, padding_top=5))
 
+        # Data to populate the table.
+        if toga.platform.current_platform in ["android", "ios"]:
+            table_data = bee_movies[:4]
+        else:
+            table_data = bee_movies*1000
```

Comment:
The demo deliberately includes lots of data specifically to test the "can it render lots of data?" use case. I'm guessing this is a limitation of the Android implementation which is rendering every row. I'd rather see this as a fix on the Android side, truncating the amount of data that is rendered (with a warning message in the console), rather than having a special case in the example.

Reply:
Hmm...how can we find out the maximum Android will render without running into the JNI restriction?

Reply:
I created a Bug for this problem, so we can address it separately from this PR:
https://github.com/beeware/toga/issues/1392